cgi-bin/DW/Console/Command/BonusIcons.pm

```diff
@@ -85,27 +85,26 @@ sub execute {
         }
 
     } elsif ( $cmd eq 'xfer' ) {
-        my $target_u = LJ::load_user( $cmdarg );
+        my $destination_u = LJ::load_user( $cmdarg );
 
         return $self->error( 'Invalid target user.' )
-            unless $target_u;
+            unless $destination_u;
         return $self->error( 'E-mail addresses do not match.' )
-            unless $to_u->prop( 'email' ) eq $target_u->prop( 'email' ); 
+            unless $to_u->prop( 'email' ) eq $destination_u->prop( 'email' ); 
+        return $self->error( 'E-mail address is not confirmed.' )
+            unless $to_u->is_validated && $destination_u->is_validated;
```

Comment:
@rahaeli, are you happy with this, or would you like the error message to say which account(s) do(es)n't have the e-mail address confirmed?